roles/kubernetes/node/templates/kubelet.service.j2

```diff
@@ -14,7 +14,7 @@ Wants={{ kubelet_dependency }}
 {% endfor %}
 
 [Service]
-EnvironmentFile=-{{ kube_config_dir }}/kubelet.env
```

Comment:
1. Please use `master` branch to merge.
2. This is not typo, look at this [documentation](https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html).
> The argument passed should be an absolute filename or wildcard expression, optionally prefixed with "-", which indicates that if the file does not exist, it will not be read and no error or warning message is logged. This option may be specified more than once in which case all specified files are read. If the empty string is assigned to this option, the list of file to read is reset, all prior assignments have no effect.

Reply:
Don't include it, the mistake was mine.